.github/workflows/rust.yml

```diff
@@ -1,3 +1,4 @@
+---
```

Comment:
What does this do?

Reply:
this marks a document start... 

https://yamllint.readthedocs.io/en/stable/rules.html#module-yamllint.rules.document_start

Reply:
But from what I can read there, it may or may not be present, depending on the setting. This is not something required by github, right?